cpp/src/precompiled/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ foreach(SRC_FILE ${PRECOMPILED_SRCS})
   add_custom_command(
     OUTPUT ${BC_FILE}
     COMMAND ${CLANG_EXECUTABLE}
-            -std=c++11 -emit-llvm -O2 -c ${ABSOLUTE_SRC} -o ${BC_FILE}
+            -D GDV_HELPERS -std=c++11 -emit-llvm -O2 -c ${ABSOLUTE_SRC} -o ${BC_FILE}
```

Comment:
let's not do this - our pre-compiled code is all plain C. no point adding namespaces to it.

Reply:
discussed offline why we need this..